src/components/webcams/WebrtcRTSPSimpleServer.vue

```diff
@@ -0,0 +1,251 @@
+<style scoped>
```

Comment:
pls move the style tag to the end of the vue file (i'm refactoring every file, when i see this "wrong" sort)